src/sentry/api/endpoints/prompts_activity.py

```diff
@@ -92,8 +92,12 @@ def put(self, request: Request, **kwargs):
         # if project_id or organization_id in required fields make sure they exist
         # if NOT in required fields, insert dummy value so dups aren't recorded
         if "project_id" in required_fields:
-            if not Project.objects.filter(id=fields["project_id"]).exists():
-                return Response({"detail": "Project no longer exists"}, status=400)
+            if not Project.objects.filter(
```

Comment:
aside this isnt actually a high severity concern afaict, but no idea what the consequence is